src/highlight/HighlightAnnotations.tsx

```diff
@@ -9,8 +9,8 @@ import PopupHighlight from '../components/Popups/PopupHighlight';
 import PopupReply from '../components/Popups/PopupReply';
 import { AnnotationHighlight } from '../@types';
 import { CreateArg } from './actions';
-import { CreatorItemHighlight, CreatorStatus, Mode, SelectionArg, SelectionItem } from '../store';
-import { getBoundingRect } from './highlightUtil';
+import { CreatorItemHighlight, CreatorStatus, Mode, SelectionItem } from '../store';
+import { getBoundingRect, getShapeRelativeToPage } from './highlightUtil';
```

Comment:
Maybe this should be `getShapeRelativeToContainer` to make it less doc-specific?

Reply:
Make sense. I can move it to `src/utils/util.ts` then.

Reply:
We're trying to faze out `util.ts`

Reply:
Discussed offline. OK I'll rename it and just leave it in `highlightUtil`.